errors.go

```diff
@@ -0,0 +1,8 @@
+package wrapping
+
+import (
+	"errors"
+)
+
+// ErrInvalidParameter represents an invalid parameter error
+var ErrInvalidParameter = errors.New("invalid parameter")
```

Comment:
Can you either put this in const.go with the other error or can you move that one in here too?